application/Commands/Layouts/Import.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace OTGS\Toolset\CLI\Layouts;
+
+use WPDD_Layouts_Theme ;
+
+/*
+ * FIXME: replace \WP_CLI with WpCli.
+ */
+
+/**
+ * Layouts import command.
+ */
+class LayoutsImport extends LayoutsCommand {
```

Comment:
Ditto regarding the class name

```suggestion
class Import extends LayoutsCommand {
```